.github/workflows/docker.yml

```
@@ -7,12 +7,12 @@ name: Docker Build & Push
 
 on:
   release:
-    types: [published]
+    types: [published, main, mainented, develop]
```

actionlint (1.7.12)

[error] invalid activity type "main" for "release" Webhook event. available types are "created", "deleted", "edited", "prereleased", "published", "released", "unpublished"

(events)